src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -506,4 +506,39 @@ public function frontendDevDisable($siteDir = 'default', array $opts = ['yes|y'
             ->remove($this->devServicesPath)
             ->run();
     }
+
+    /**
+     * Setup a site in GitHub Codespaces.
+     *
+     * @throws \Robo\Exception\TaskException
+     */
+    public function setupCodespaces()
+    {
+        $codespaces_directory = getenv('PWD') . '/web';
+        if (empty($codespaces_directory)) {
+            throw new TaskException($this, 'Codespaces directory is unavailable.');
+        }
+        $result = $this->taskExec('rm /var/www/html')->run();
```

Comment:
There are commands to do some of this stuff in a more "robo way" FWIW.

https://robo.li/tasks/Filesystem/